assets/svelte/components/PageAstNode.svelte

```diff
@@ -130,27 +137,6 @@
     }
   }
 
-  function bindIfSelected(el: HTMLElement, isSelected: boolean) {
-    if (isSelected) {
-      $selectedDomElement = el
-      initSelectedElementDragMenuPosition(el)
-    }
-
-    return {
-      update(isSelected) {
-        if (isSelected) {
-          $selectedDomElement = el
-          initSelectedElementDragMenuPosition(el)
-        }
-      },
-      destroy() {
-        if (isSelected) {
-          $selectedDomElement = null
-        }
-      },
-    }
-  }
-
```

Comment:
We didn't need this bind anymore with the updated `handleClick` function. Everything still seems to work the same on showing the drag-menu in the selected element